README.md

```diff
@@ -1,2 +1,14 @@
 # imago
 An picture organization software for managing pictures in multiple locations both local and web based.
+
+
+### Building using Haxe
+
```

Comment:
In general, the projects I have worked in on GitHub use the README.md for dev/contributor setup information, as well as the project overview and new user basics.  I stuck with that here.